gremlin-javascript/src/main/javascript/gremlin-javascript/test/integration/sasl-authentication-tests.js

```diff
@@ -69,38 +68,43 @@ describe('DriverRemoteConnection', function () {
 
       it('should return error when using ws:// for a TLS configured server', function () {
         const authenticator = new PlainTextSaslAuthenticator('stephen', 'password');
-        connection =  helper.getDriverRemoteConnection(badServerAuthUrl, {
+        connection = getDriverRemoteConnection(badServerAuthUrl, {
           authenticator: authenticator,
-          rejectUnauthorized: false
+          rejectUnauthorized: false,
         });
 
-        return connection.submit(new Bytecode().addStep('V', []).addStep('tail', []))
-            .then(function() {
-              assert.fail("server is running TLS and trying to connect with ws:// so this should result in error thrown");
-            })
-            .catch(function (err) {
-              if (err instanceof AssertionError) throw err;
-              assert.ok(err);
-              assert.ok(err.message === 'socket hang up');
-            });
+        return connection
+          .submit(new Bytecode().addStep('V', []).addStep('tail', []))
+          .then(function () {
+            assert.fail('server is running TLS and trying to connect with ws:// so this should result in error thrown');
+          })
+          .catch(function (err) {
+            if (err instanceof AssertionError) throw err;
+            assert.ok(err);
+            assert.ok(err.message === 'socket hang up');
+          });
       });
 
       it('should return error when using ws:// for a TLS configured server', function () {
         const authenticator = new PlainTextSaslAuthenticator('stephen', 'password');
-        connection =  helper.getDriverRemoteConnection(badServerAuthUrl, {
+        connection = getDriverRemoteConnection(badServerAuthUrl, {
           authenticator: authenticator,
-          rejectUnauthorized: false
+          rejectUnauthorized: false,
         });
 
-        const g = traversal().with_(connection);
-        return g.V().toList().then(function() {
-          assert.fail("server is running TLS and trying to connect with ws:// so this should result in error thrown");
-        }).catch(function(err) {
-        if (err instanceof AssertionError) throw err;
-              assert.ok(err);
-              assert.ok(err.message === 'socket hang up');
+        const g = traversal().withRemote(connection);
```

Comment:
`withRemote` will be depricated in 4.0, `with_(` should be used.
the same thing a few more times

Reply:
Oops, must have been brought over since I originally did this on the 3.7 branch. Have replaced all `withRemote` with `with_`